http-client/src/test/groovy/io/micronaut/http/client/ClientIntroductionAdviceSpec.groovy

```diff
@@ -132,6 +158,35 @@ class ClientIntroductionAdviceSpec extends Specification {
         }
     }
 
+    @Requires(property = 'spec.name', value = 'ClientIntroductionAdviceSpec')
+    @Controller('/accept')
+    static class AcceptTypeController {
+
+        @Get
+        String index(HttpRequest<?> request) {
+            if (!request.accept().contains(MediaType.APPLICATION_JSON_TYPE)) {
+                throw new IllegalStateException("Accept should default to JSON")
+            }
+            return "success"
+        }
+
+        @Get(value = "/xml", produces = MediaType.APPLICATION_XML)
+        String xml(HttpRequest<?> request) {
+            if (!request.accept().contains(MediaType.APPLICATION_XML_TYPE)) {
+                throw new IllegalStateException("Accept should be set to XML")
+            }
+            return "<answer>success</answer>"
+        }
+
+        @Get(value = "/xml-annotated", produces = MediaType.APPLICATION_XML)
+        String xmlAnnotated(HttpRequest<?> request) {
```

Comment:
Delete this method.

Reply:
Done in [6c3106b](https://github.com/micronaut-projects/micronaut-core/pull/10685/commits/6c3106b2a3d55b0587eb04188547d10d5980fe63)